cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -54,7 +54,7 @@
       cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','run_server_generate_curriculum_pdfs')
       cronjob at:'*/2 * * * *', do:pegasus_dir('sites','virtual','collate_pdfs')
       cronjob at:'30 16,17 * * *', do:deploy_dir('bin', 'cron', 'update_dotd')
-      cronjob at:'0 * * * *', do:deploy_dir('tools', 'scripts', 'brokenLinkChecker', 'brokenLinkChecker.js'), notify:'dev+crontab@code.org'
+      cronjob at:'0 * * * *', do:deploy_dir('bin', 'cron', 'start_broken_link_checker'), notify:'dev+crontab@code.org'
```

Comment:
I don't think this line matches the `start_broken_link_checker.rb` file added in this PR (`.rb` missing). Should one or the other be renamed?

Reply:
Oops, will fix now.  Thanks!